hotline/species/forms.py

```diff
@@ -0,0 +1,72 @@
+from django import forms
+from django.db.models import Q
+from django.utils.safestring import mark_safe
+from elasticmodels.forms import SearchForm
+
+from .indexes import SpeciesIndex
+
+
+class SpeciesSearchForm(SearchForm):
+    """
+    This form handles searching for a species in the species list view.
+    """
+    q = None
+
+    querystring = forms.CharField(required=False, widget=forms.widgets.TextInput(attrs={
+        "placeholder": "name:Foobarius"
+    }), label=mark_safe("Search <a target='_blank' class='help' href='help'>[?]</a>"))
+
+    sort_by = forms.ChoiceField(choices=[
+        ("name", "Name"),
+        ("scientific_name", "Scientific Name"),
+        ("severity", "Severity"),
+        ("category", "Category"),
+        ("is_confidential", "Confidential"),
+    ], required=False)
+
+    order = forms.ChoiceField(choices=[
+        ("ascending", "Ascending"),
+        ("descending", "Descending"),
+    ], required=False, initial="ascending", widget=forms.widgets.RadioSelect)
+
+    def __init__(self, *args, user, species_ids=(), **kwargs):
+        self.user = user
+        self.species_ids = species_ids
```

Comment:
I don't think you need this species_ids business anywhere. The reason that report_ids stuff exists on the ReportSearchForm has to do with anonymous users who submit reports need to be able to view them, despite the fact that the report is non-public (if that makes sense).